code/game/machinery/doors/brigdoors.dm

```diff
@@ -63,6 +63,24 @@
 			stat |= BROKEN
 		update_icon()
 
+/obj/machinery/door_timer/proc/print_report()
+	var/logname = input(usr, "Name of the guilty?","[id] log name")
+	var/logcharges = input(usr, "What are they being charged with?","[id] log charges")
+
+	for(var/obj/machinery/computer/prisoner/C in prisoncomputer_list)
+
+		var/obj/item/weapon/paper/P = new /obj/item/weapon/paper(C.loc)
+		P.name = "[id] log - [logname]"
+		P.info =  "<center><b>[id] - Brig record</b></center><br><hr><br>"
+		P.info += "<center>NSS Cyberaid - Security Department</center><br>"
+		P.info += "<center><small><b>Admission data:</b></small></center><br>"
+		P.info += "<small><b>Log generated at:   </b>		[worldtime2text(world.time)]<br>"
+		P.info += "<b>Suspect:</b>		[logname]<br>"

```

Comment:
Instead of "Suspect", shouldn't this be "Detainee" or something, since you (hopefully) have enough evidence to justify them being incarcerated and thus they aren't merely a suspect?